.github/ISSUE_TEMPLATE/2-enhancement.yml

```diff
@@ -0,0 +1,33 @@
+name: ✨ Enhancement Request
+description: If you have an idea to improve an existing feature or need something for development (such as a new hook) please let us know or submit a Pull Request!
+title: "[Enhancement]: "
+labels: ["type: enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for contributing!
+
+        Please provide us with the information requested in this form. 
+
+        Make sure to look through [existing `type: enhancement` issues](https://github.com/Automattic/wc-calypso-bridge/issues?q=is%3Aopen+is%3Aissue+label%3A%22type%3A+enhancement%22) and [existing `votes needed` issues](https://github.com/Automattic/wc-calypso-bridge/issues?q=is%3Aissue+sort%3Areactions-%2B1-desc+label%3A%22needs%3A+votes%22+) to see whether your idea is already being discussed.
```

Comment:
`votes needed` label isn't available here. Perhaps we can remove the sentence altogether?

Reply:
Sure. Removed in 89181a9b726ab11b9f17da5039605b1c9d38b26c